pkg/common/events/states.go

```diff
@@ -53,6 +53,7 @@ type ApplicationStates struct {
 	Failing    string
 	Failed     string
 	Resuming   string
+	Any        []string // Any refers to all possible states
```

Comment:
Sorry for this one, as I can se we are not using it. I mixed it up with the Task state, where we are using it, and during my debugging I thought that the transition is broken because of that one. If you agree that we don't need it you can remove it.